frontend/public/manifest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://json.schemastore.org/web-manifest",
+  "name": "Universal BUK Games",
+  "short_name": "UBG",
+  "description": "The Universal BUK Games on your device",
+  "lang": "en",
+  "start_url": ".",
+  "scope": "/",
+  "display": "fullscreen",
```

Comment:
standalone (the default) might be better, running as it's own app but allow notification bar etc to still show.

Reply:
I based my choice on the [MDN `display` documentation](https://developer.mozilla.org/en-US/docs/Web/Manifest/display). Fullscreen should refer here to the display of the PWA within the browser, not the PWA in the OS.

> `fullscreen` All of the available display area is used and no user agent [chrome](https://developer.mozilla.org/en-US/docs/Glossary/Chrome) is shown.

I based it on the fact that they explicitly mention the _chrome_ (*The* chrome explicitly refers to Browser UI elements), and the fact that they state reference _all of the **available** display area_ made me think that `fullscreen` does not hide the OS title bar, and does not mean a full-screen native application.

This [this stackoverflow answer](https://stackoverflow.com/questions/33129326/chrome-web-app-manifest-differences-between-display-types) however, states that `standalone` is what we desire, and `fullscreen` hides the status bar.

Note that this is _not true_ on my desktop. There a PWA installed with `fullscreen` mode behaves as expected. It is not fullscreen, and there's (almost) no visible chrome.

Reply:
Phone is definitely the place to test. Fullscreen is also ignored on my desktop, but that might be an adjustment on the desktop side of things. I think standalone is the best match for now.

Reply:
Standalone seems to work best indeed.